CMakeLists.txt

```diff
@@ -329,13 +329,13 @@ endif()
 
 # add the path to detect Ogre3D
 if(WIN32)
-  list(APPEND CMAKE_MODULE_PATH "${OGRE_DIR}/cmake")
-  list(APPEND CMAKE_MODULE_PATH "${OGRE_DIR}/CMake")
+  list(APPEND CMAKE_MODULE_PATH "${OGRE_DIR}")
+  list(APPEND CMAKE_MODULE_PATH "${OGRE_DIR}")
```

Comment:
Hi, thanks for this PR.
I would prefer that you add a new path instead of replacing that could lead to side effects.
Your 2 lines are the same.

Since windows is unable to make difference between lower and upper case, put simply 
```
list(APPEND CMAKE_MODULE_PATH "${OGRE_DIR}/cmake")
list(APPEND CMAKE_MODULE_PATH "${OGRE_DIR}")
```

Reply:
My bad, did not paid attention that it lead to duplicated lines.
Ok to me to keep both `OGRE_DIR` paths (with and without the `cmake` suffix).